Unable to build Quarkus on "non developer" machine with just JDK installed

### Describe the bug

I was unable to build Quarkus on my wife's (non developer) Windows machine which has just JDK installed

The build ended in few seconds with 
```
[INFO] Maveniverse Nisse 0.4.1 loaded
[ERROR] Exception in JGitPropertySource: java.lang.IllegalArgumentException: One of setGitDir or setWorkTree must be called.
[ERROR] Error executing Maven.
[ERROR] java.lang.IllegalArgumentException: One of setGitDir or setWorkTree must be called.
[ERROR] Caused by: One of setGitDir or setWorkTree must be called.
```

### Expected behavior

Require just JDK to build Quarkus

### Actual behavior

Maveniverse Nisse extension expects that the repository was git cloned, but in non-dev environment the usual flow is to just download the ZIP from the repository.

### How to Reproduce?

```
wget https://github.com/quarkusio/quarkus/archive/refs/heads/main.zip
unzip -q main.zip
cd quarkus-main
./mvnw install -Dquickly 
```

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/.mvn/extensions.xml
+++ b/.mvn/extensions.xml
@@ -22,6 +22,6 @@
     <extension>
         <groupId>eu.maveniverse.maven.nisse</groupId>
         <artifactId>extension</artifactId>
-        <version>0.4.1</version>
+        <version>0.4.2</version>
     </extension>
 </extensions>

